spec/support/stubs/appsignal/loaders/loader_stub.rb

```diff
@@ -0,0 +1,7 @@
+module Appsignal
+  module Loaders
+    class LoaderStub < Loader
+      register :loader_stub
+    end
+  end
+end
```

Comment:
Are both this and `LoaderTester` (in `loader_tester.rb`) needed? They seem identical to me, I think the tests that use `LoaderStub` could use `LoaderTester` or vice versa. I may not be understanding why both exist.

Reply:
I use `loader_stub.rb` to test if it requires the file. That can only be used once, because if I use it in another test and it gets called before that test, it doesn't work anymore.

I use `loader_test.rb` to test if something is an instance of a named class. It doesn't work with anonymous classes. I think I can change it so it's a bit less confusing.